tools/genevergreen/generate/generate.go

```diff
@@ -191,7 +199,7 @@ func PublishStableTasks(c *shrub.Configuration, toolName string) {
 	}
 	for _, sv := range serverVersions {
 		v := &shrub.Variant{
-			BuildName:        fmt.Sprintf("generated_release_%s_publish_%s", toolName, sv),
+			BuildName:        fmt.Sprintf("generated_release_%s_publish_%s", toolName, strings.ReplaceAll(sv, ".", "")),
```

Comment:
removing the `.` from the variant name. From `test_repo_atlascli_amazonlinux2_org_4.2 `to `test_repo_atlascli_amazonlinux2_org_42 ` as it was before https://github.com/mongodb/mongodb-atlas-cli/pull/1448